ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/AttestationManager.java

```diff
@@ -197,7 +197,10 @@ private void applyFutureAttestations(final UInt64 slot) {
   }
 
   @Override
-  public void onBlockImported(final SignedBeaconBlock block) {
+  public void onBlockValidated(final SignedBeaconBlock block) {}
```

Comment:
Nit: 
Im happy with this kind of format, but above we've used `//noop` in the function...

stylistically can probably just pick one format and stick with that....